modules/apps/portal-language/portal-language-extender/src/main/java/com/liferay/portal/language/extender/internal/LanguageResourcesExtender.java

```diff
@@ -144,10 +144,13 @@ private void _registerResourceBundles(
 
 			String languageId = StringPool.BLANK;
 
-			if (urlPath.contains(StringPool.UNDERLINE)) {
+			index = StringUtil.indexOfAny(
```

Comment:
@yuhai it is a single char searching, why `StringUtil.indexOfAny()` please keep it as plain String.indexOf() for char.